src/commonComponents/AppWrapper/AppWrapper.jsx

```diff
@@ -11,36 +12,40 @@ function AppWrapper() {
   //       setData(data);
```

Comment:
please, avoid using commented parts of code in the PRs

Reply:
Cleared all comments